sources/Interop/D3D12/um/d3dx12/D3D12.Manual.cs

```diff
@@ -23,6 +23,28 @@ public static unsafe partial class D3D12
             Count = 1,
             Quality = 0,
         };
+        
+        [return: NativeTypeName("HRESULT")]
+        public static int CreateRootSignature([NativeTypeName("ID3D12Device*")] ID3D12Device* device, [NativeTypeName("const D3D12_ROOT_SIGNATURE_DESC*")] D3D12_ROOT_SIGNATURE_DESC* rootSignatureDesc, [NativeTypeName("ID3D12RootSignature**")] ID3D12RootSignature** rootSignature)
```

Comment:
Clang normalizes to have a space between the type and the `*`
```suggestion
        public static int CreateRootSignature([NativeTypeName("ID3D12Device *")] ID3D12Device* device, [NativeTypeName("const D3D12_ROOT_SIGNATURE_DESC *")] D3D12_ROOT_SIGNATURE_DESC* rootSignatureDesc, [NativeTypeName("ID3D12RootSignature **")] ID3D12RootSignature** rootSignature)
```